src/stores/messages/MessagesDataProviderStore.ts

```diff
@@ -109,6 +110,7 @@ export default class MessagesDataProviderStore {
 	@action
 	public loadMessages = async () => {
 		this.stopMessagesLoading();
+		this.resetMessagesDataState();
```

Comment:
since we reseting state here now, can we delete this function call in `onFilterChange`